src/index.js

```diff
@@ -13,9 +13,11 @@ const dateClass = mockDateClass(Date);
 if (global.window) {
   // dom env
   global.window.Date = dateClass;
+  global.window.performance.now = function() { return dateClass.now(); };
 } else {
   // node / native env
   global.Date = dateClass;
+  require('perf_hooks').performance.now = function() { return dateClass.now(); };
```

Comment:
what is perf_hooks?

Reply:
It's a performance measurement API built into node's API. It's essentially node's version of the browser's monotonic clock (window.performance.now).

When mocking the non-monotonic clock I think it's unnatural for the monotonic clock to be free.